Kiryuu: `Too many follow-up requests` crash

### Source information

Kiryuu 1.4.48

### Source language

Id

### Steps to reproduce

1. Click the Kiryuu extension crashes in other apps on Mihon as shown in the picture


Issue:
1. Click extension kiryuu 
2. Click latest and crash

### Expected behavior

Fix and no crash

### Actual behavior

I clicked on the Kiryuu extension, then clicked on the latest version, and then a crash appeared, as shown in the image and log.

### Mihon/Mihon fork version

Mihon 1.9.5

### Android version

Android 11

### Other details

App ID: app.mihon
App version: 0.19.5 (b58d2a5, 19, 2026-03-20T09:32:32Z)
Installation ID: b969f1eb-814d-48eb-9829-4212562927d4
Android version: 11 (SDK 30; build RMX2103_11_C.17)
Device brand: realme
Device manufacturer: realme
Device name: RE50C1 (RMX2103T2)
Device model: RMX2103
WebView: Android System WebView 146.0.7680.164
Current time: 2026-04-04T17:17:33.938+07:00

java.lang.Throwable: java.io.IOException: Too many follow-up requests: 21
	at eu.kanade.tachiyomi.multisrc.natsuid.NatsuId$getFilterList$1.invokeSuspend(NatsuId.kt:155)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:9)
	at kotlinx.coroutines.DispatchedTask.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:109)
	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:4)
	at kotlinx.coroutines.scheduling.TaskImpl.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:3)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:1)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:58)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [androidx.compose.runtime.PausableMonotonicFrameClock@d6b190c, androidx.compose.ui.platform.MotionDurationScaleImpl@3838255, StandaloneCoroutine{Cancelling}@372946a, AndroidUiDispatcher@4fdad5b]
Caused by: java.net.ProtocolException: Too many follow-up requests: 21
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:149)
	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
	at okhttp3.CompressionInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:46)
	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
	at eu.kanade.tachiyomi.network.interceptor.RateLimitInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:142)
	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
	at eu.kanade.tachiyomi.network.interceptor.WebViewInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:2)
	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
	at eu.kanade.tachiyomi.network.interceptor.UserAgentInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:25)
	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
	at eu.kanade.tachiyomi.network.interceptor.UncaughtExceptionInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:10)
	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:90)
	at okhttp3.internal.connection.RealCall$AsyncCall.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:44)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
	at java.lang.Thread.run(Thread.java:923)


--------- beginning of system
--------- beginning of main
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: Error was captured in composition.
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: java.io.IOException: Too many follow-up requests: 21
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at eu.kanade.tachiyomi.multisrc.natsuid.NatsuId$getFilterList$1.invokeSuspend(NatsuId.kt:155)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:9)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at kotlinx.coroutines.DispatchedTask.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:109)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:4)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at kotlinx.coroutines.scheduling.TaskImpl.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:3)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:1)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:58)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: Caused by: java.net.ProtocolException: Too many follow-up requests: 21
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:149)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at okhttp3.CompressionInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:46)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at eu.kanade.tachiyomi.network.interceptor.RateLimitInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:142)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at eu.kanade.tachiyomi.network.interceptor.WebViewInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:2)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at eu.kanade.tachiyomi.network.interceptor.UserAgentInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:25)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at eu.kanade.tachiyomi.network.interceptor.UncaughtExceptionInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:10)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:90)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at okhttp3.internal.connection.RealCall$AsyncCall.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:44)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
2026-04-04 17:17:03.064 +0700 24575 24575 E ComposeInternal: 	at java.lang.Thread.run(Thread.java:923)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: java.io.IOException: Too many follow-up requests: 21
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at eu.kanade.tachiyomi.multisrc.natsuid.NatsuId$getFilterList$1.invokeSuspend(NatsuId.kt:155)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:9)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at kotlinx.coroutines.DispatchedTask.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:109)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:4)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at kotlinx.coroutines.scheduling.TaskImpl.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:3)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:1)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:58)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [androidx.compose.runtime.PausableMonotonicFrameClock@5786830, androidx.compose.ui.platform.MotionDurationScaleImpl@9232ca9, StandaloneCoroutine{Cancelling}@f4bb22e, AndroidUiDispatcher@11911cf]
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: Caused by: java.net.ProtocolException: Too many follow-up requests: 21
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:149)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.CompressionInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:46)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at eu.kanade.tachiyomi.network.interceptor.RateLimitInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:142)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at eu.kanade.tachiyomi.network.interceptor.WebViewInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:2)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at eu.kanade.tachiyomi.network.interceptor.UserAgentInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:25)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at eu.kanade.tachiyomi.network.interceptor.UncaughtExceptionInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:10)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:90)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.connection.RealCall$AsyncCall.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:44)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
2026-04-04 17:17:03.072 +0700 24575 24575 E GlobalExceptionHandler: 	at java.lang.Thread.run(Thread.java:923)
--------- beginning of crash
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: FATAL EXCEPTION: main
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: Process: app.mihon, PID: 24575
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: java.io.IOException: Too many follow-up requests: 21
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at eu.kanade.tachiyomi.multisrc.natsuid.NatsuId$getFilterList$1.invokeSuspend(NatsuId.kt:155)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:9)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at kotlinx.coroutines.DispatchedTask.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:109)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:4)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.TaskImpl.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:3)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:1)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:58)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [androidx.compose.runtime.PausableMonotonicFrameClock@5786830, androidx.compose.ui.platform.MotionDurationScaleImpl@9232ca9, StandaloneCoroutine{Cancelling}@f4bb22e, AndroidUiDispatcher@11911cf]
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: Caused by: java.net.ProtocolException: Too many follow-up requests: 21
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:149)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at okhttp3.CompressionInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:46)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at eu.kanade.tachiyomi.network.interceptor.RateLimitInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:142)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at eu.kanade.tachiyomi.network.interceptor.WebViewInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:2)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at eu.kanade.tachiyomi.network.interceptor.UserAgentInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:25)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at eu.kanade.tachiyomi.network.interceptor.UncaughtExceptionInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:10)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:90)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at okhttp3.internal.connection.RealCall$AsyncCall.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:44)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
2026-04-04 17:17:03.300 +0700 24575 24575 E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:923)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: java.io.IOException: Too many follow-up requests: 21
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at eu.kanade.tachiyomi.multisrc.natsuid.NatsuId$getFilterList$1.invokeSuspend(NatsuId.kt:155)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:9)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at kotlinx.coroutines.DispatchedTask.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:109)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:4)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at kotlinx.coroutines.scheduling.TaskImpl.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:3)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:1)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:58)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [androidx.compose.runtime.PausableMonotonicFrameClock@5786830, androidx.compose.ui.platform.MotionDurationScaleImpl@9232ca9, StandaloneCoroutine{Cancelling}@f4bb22e, AndroidUiDispatcher@11911cf]
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: Caused by: java.net.ProtocolException: Too many follow-up requests: 21
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:149)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.CompressionInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:46)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at eu.kanade.tachiyomi.network.interceptor.RateLimitInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:142)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at eu.kanade.tachiyomi.network.interceptor.WebViewInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:2)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at eu.kanade.tachiyomi.network.interceptor.UserAgentInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:25)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at eu.kanade.tachiyomi.network.interceptor.UncaughtExceptionInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:10)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:90)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at okhttp3.internal.connection.RealCall$AsyncCall.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:44)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
2026-04-04 17:17:03.300 +0700 24575 24575 E GlobalExceptionHandler: 	at java.lang.Thread.run(Thread.java:923)
2026-04-04 17:17:03.548 +0700 26066 26066 E n:error_handle: Unknown bits set in runtime_flags: 0x40000000
2026-04-04 17:17:03.553 +0700 26066 26066 E n:error_handle: Not starting debugger since process cannot load the jdwp agent.
2026-04-04 17:17:03.558 +0700 26066 26066 E RefClass: java.lang.reflect.InvocationTargetException
2026-04-04 17:17:03.605 +0700 26066 26066 E libc    : Access denied finding property "ro.odm.prev.product.name"
2026-04-04 17:17:03.847 +0700 26066 26120 E libEGL  : Invalid file path for libcolorx-loader.so
2026-04-04 17:17:03.989 +0700 26066 26066 E MultiApp.Impl: OplusMultiAppImpl
2026-04-04 17:17:04.337 +0700 26153 26153 E app.mihon: Unknown bits set in runtime_flags: 0x40000000
2026-04-04 17:17:04.341 +0700 26153 26153 E app.mihon: Not starting debugger since process cannot load the jdwp agent.
2026-04-04 17:17:04.347 +0700 26153 26153 E RefClass: java.lang.reflect.InvocationTargetException
2026-04-04 17:17:04.370 +0700 26153 26153 E libc    : Access denied finding property "ro.odm.prev.product.name"
2026-04-04 17:17:04.477 +0700 26066 26118 E libEGL  : Invalid file path for libcolorx-loader.so
2026-04-04 17:17:04.507 +0700 26066 26118 E libEGL  : Invalid file path for libcolorx-loader.so
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: FATAL EXCEPTION: main
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: Process: app.mihon, PID: 26153
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: java.io.IOException: Too many follow-up requests: 21
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at eu.kanade.tachiyomi.multisrc.natsuid.NatsuId$getFilterList$1.invokeSuspend(NatsuId.kt:155)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:9)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at kotlinx.coroutines.DispatchedTask.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:109)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:4)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.TaskImpl.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:3)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:1)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:58)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [androidx.compose.runtime.PausableMonotonicFrameClock@21738f5, androidx.compose.ui.platform.MotionDurationScaleImpl@9bc168a, StandaloneCoroutine{Cancelling}@cafd4fb, AndroidUiDispatcher@ee45918]
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: Caused by: java.net.ProtocolException: Too many follow-up requests: 21
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:149)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at okhttp3.CompressionInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:46)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at eu.kanade.tachiyomi.network.interceptor.RateLimitInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:142)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at eu.kanade.tachiyomi.network.interceptor.WebViewInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:2)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at eu.kanade.tachiyomi.network.interceptor.UserAgentInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:25)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at eu.kanade.tachiyomi.network.interceptor.UncaughtExceptionInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:10)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:90)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at okhttp3.internal.connection.RealCall$AsyncCall.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:44)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
2026-04-04 17:17:18.874 +0700 26153 26153 E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:923)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: FATAL EXCEPTION: main
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: Process: app.mihon, PID: 26361
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: java.io.IOException: Too many follow-up requests: 21
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at eu.kanade.tachiyomi.multisrc.natsuid.NatsuId$getFilterList$1.invokeSuspend(NatsuId.kt:155)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:9)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at kotlinx.coroutines.DispatchedTask.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:109)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:4)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.TaskImpl.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:3)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:1)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:58)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [androidx.compose.runtime.PausableMonotonicFrameClock@d6b190c, androidx.compose.ui.platform.MotionDurationScaleImpl@3838255, StandaloneCoroutine{Cancelling}@372946a, AndroidUiDispatcher@4fdad5b]
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: Caused by: java.net.ProtocolException: Too many follow-up requests: 21
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:149)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at okhttp3.CompressionInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:46)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at eu.kanade.tachiyomi.network.interceptor.RateLimitInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:142)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at eu.kanade.tachiyomi.network.interceptor.WebViewInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:2)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at eu.kanade.tachiyomi.network.interceptor.UserAgentInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:25)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at eu.kanade.tachiyomi.network.interceptor.UncaughtExceptionInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:10)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:90)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at okhttp3.internal.connection.RealCall$AsyncCall.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:44)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
2026-04-04 17:17:30.190 +0700 26361 26361 E AndroidRuntime: 	at java.lang.Thread.run(Thread.java:923)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: java.io.IOException: Too many follow-up requests: 21
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at eu.kanade.tachiyomi.multisrc.natsuid.NatsuId$getFilterList$1.invokeSuspend(NatsuId.kt:155)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:9)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at kotlinx.coroutines.DispatchedTask.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:109)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:4)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at kotlinx.coroutines.scheduling.TaskImpl.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:3)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:1)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:58)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [androidx.compose.runtime.PausableMonotonicFrameClock@d6b190c, androidx.compose.ui.platform.MotionDurationScaleImpl@3838255, StandaloneCoroutine{Cancelling}@372946a, AndroidUiDispatcher@4fdad5b]
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: Caused by: java.net.ProtocolException: Too many follow-up requests: 21
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:149)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at okhttp3.CompressionInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:46)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at eu.kanade.tachiyomi.network.interceptor.RateLimitInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:142)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at eu.kanade.tachiyomi.network.interceptor.WebViewInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:2)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at eu.kanade.tachiyomi.network.interceptor.UserAgentInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:25)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at eu.kanade.tachiyomi.network.interceptor.UncaughtExceptionInterceptor.intercept(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:10)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at okhttp3.internal.http.RealInterceptorChain.proceed(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:133)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at okhttp3.internal.connection.RealCall.getResponseWithInterceptorChain$okhttp(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:90)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at okhttp3.internal.connection.RealCall$AsyncCall.run(r8-map-id-680f9471a8e1de12c1725c413132c1c28ae3592559db9c1f01303dd6c1713c5f:44)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1167)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:641)
2026-04-04 17:17:30.190 +0700 26361 26361 E GlobalExceptionHandler: 	at java.lang.Thread.run(Thread.java:923)
2026-04-04 17:17:30.369 +0700 26066 26477 E libEGL  : Invalid file path for libcolorx-loader.so

![Image](https://github.com/user-attachments/assets/6c26d7a6-b64e-4674-8ab4-00cc85d1cb07)

### Acknowledgements

- [x] I have searched [existing issues](https://github.com/keiyoushi/extensions-source/issues) both open & closed, and confirm that this is a new unreported issue.
- [x] I have written a short but informative title.
- [x] I have updated the app to version **[0.19.3](https://github.com/mihonapp/mihon/releases/latest)**.
- [x] I have updated the extension to the latest version (as listed on the [extensions listing](https://keiyoushi.github.io/extensions/) page).
- [x] I have tried the [troubleshooting guide](https://mihon.app/docs/guides/troubleshooting/).
- [x] If this is an issue with the app itself, I should be opening an issue in the app repository.
- [x] I will correctly fill out all of the requested information in this form.

### <!-- footer -->


---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/keiyoushi/extensions-source/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/src/id/kiryuu/build.gradle
+++ b/src/id/kiryuu/build.gradle
@@ -2,8 +2,8 @@ ext {
     extName = 'Kiryuu'
     extClass = '.Kiryuu'
     themePkg = 'natsuid'
-    baseUrl = 'https://v1.kiryuu.to'
-    overrideVersionCode = 46
+    baseUrl = 'https://v2.kiryuu.to'
+    overrideVersionCode = 47
     isNsfw = false
 }
 

--- a/src/id/kiryuu/src/eu/kanade/tachiyomi/extension/id/kiryuu/Kiryuu.kt
+++ b/src/id/kiryuu/src/eu/kanade/tachiyomi/extension/id/kiryuu/Kiryuu.kt
@@ -11,7 +11,7 @@ class Kiryuu :
     NatsuId(
         "Kiryuu",
         "id",
-        "https://v1.kiryuu.to",
+        "https://v2.kiryuu.to",
     ) {
     // Formerly "Kiryuu (WP Manga Stream)"
     override val id = 3639673976007021338

